docs/manual/moved/associations.md

```diff
@@ -0,0 +1,16 @@
+# \[MOVED\] Associations
+
+After a major rewrite in Sequelize docs, the contents of this page were spread around other guides.

```

Comment:
We don't want to say that :) Perhaps something like this

```
Contents of this page is now available as other specialized guides.

If you're here, you might be looking for these topics:
```

Please improve this if you want to, no need to mention major rewrite in docs :) Please replace this message in other docs as well

Reply:
[Fixed](https://github.com/sequelize/sequelize/pull/11825/commits/23d1a42d999b256a75bbf975db201d86dca1b5dd)